core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
+                              refresher_exploration_id: null
+                            },
+                            rule_specs: [],
+                          }],
+                          confirmed_unclassified_answers: [],
+                          customization_args: {
+                            placeholder: {
+                              value: {
+                                content_id: 'ca_placeholder_0',
+                                unicode_str: ''
+                              }
+                            },
+                            rows: { value: 1 },
+                            catchMisspellings: {
+                              value: false
+                            }
+                          },
+                          default_outcome: {
+                            dest: null,
+                            dest_if_really_stuck: null,
+                            feedback: {
+                              html: 'Correct Answer',
+                              content_id: 'content_2'
+                            },
+                            param_changes: [],
+                            labelled_as_correct: true
+                          },
+                          hints: [{
+                            hint_content: {
+                              html: 'Hint 1',
+                              content_id: 'content_3'
+                            }
+                          }],
+                          solution: {
+                            correct_answer: 'This is the correct answer',
+                            answer_is_exclusive: false,
+                            explanation: {
+                              html: 'Solution explanation',
+                              content_id: 'content_4'
+                            }
+                          },
+                          id: 'TextInput'
+                        },
+                        param_changes: [],
+                        recorded_voiceovers: {
+                          voiceovers_mapping: {}
+                        }
+                      },
+                    },
+                    content_html: 'Translation',
+                    translation_html: 'Tradução',
+                    skill_id: 'skill_id'
+                  },
+                  status: 'rejected',
+                  exploration_content_html: null
+                },
+                details: {
+                  topic_name: 'topic_name',
+                  story_title: 'story_title',
+                  chapter_title: 'chapter_title'
+                }
+              }
+            },
+            'suggestion_1',
+            true
+            );
+        });
+        // Wait for the first test to complete.
+        tick();
+        // Now load a question contribution.
+        getUserCreatedQuestionSuggestionsAsyncSpy.and.returnValue(
+          Promise.resolve({
+            suggestionIdToDetails: {
+              suggestion_2: {
```

Comment:
Should assign this to a new variable so that we don't have to repeat it again.

Reply:
Done